src/Wallabag/CoreBundle/Resources/views/themes/material/Entry/entries.html.twig

```diff
@@ -130,6 +130,11 @@
                     {{ form_label(form.isUnread) }}
                 </div>
 
+                <div class="input-field col s6 with-checkbox">
+                    {{ form_widget(form.isAnnotated) }}
+                    {{ form_label(form.isAnnotated) }}
+                </div>
+
```

Comment:
I think we'll have issues with this label, we should put it on a new line alone.

![2021-08-07-104020_233x128](https://user-images.githubusercontent.com/226063/128594510-e102f509-18b4-415f-a8b6-a18c4cb57c2d.png)